code/game/machinery/computer/Operating.dm

```diff
@@ -13,12 +13,12 @@
 	find_table()
 
 /obj/machinery/computer/operating/proc/find_table()
-	for(var/dir in GLOB.cardinals)

```

Comment:
I'm not actually sure if `var/dir` worked as intended or not, but this makes it consistent with the rest of computer code either way.